src/main/c/Include/jep_util.h

```diff
@@ -169,4 +169,6 @@ extern jclass JDOUBLE_ARRAY_TYPE;
 #define DEFINE_CLASS_GLOBAL(var, name) extern jclass var;
 CLASS_TABLE(DEFINE_CLASS_GLOBAL)
 
+int jep_util_type_ready(PyTypeObject **target_type, PyType_Spec *spec, PyTypeObject *base);
```

Comment:
This is a nice utility. I feel like in general, the project should switch to using heap-allocated types.

That seems like it's becoming the new sort of standard for CAPI extensions.